sections/remotes.tex

```diff
@@ -234,6 +205,16 @@ \subsection{Forking Workflow - Pull Request}
 
 \end{frame}
 
+\subsection{Forking Workflow - Pull Request}
```

Comment:
I would add this slide after the next one (because the change is already merged in this graphic)